department-of-veterans-affairs/caseflow
# Extend judge reassign action to JudgeLegacyAssignTasks

References #11761

### Description
Adds the ability for judges to reassign `JudgeLegacyAssignTask`s from the case details page in an effort to remove the ability to do this from the "Assign cases" page. Assigning cases from the assigned cases page will be reserved only for assigning tasks to attorneys.

### Acceptance Criteria
- [x] Judges can reassign legacy judge tasks

### Testing Plan
1. Log in as a judge with legacy assign tasks ("BVAAABSHIRE")
2. Select a legacy appeal from their assigned cases page
3. click the task dropdown and select "Re-assign to a judge"
4. Select any other judge from the list ("Judge has case to assign") and press submit
5. Log in as that user ("BVAEBECKER") and make sure that case appears in their "Assign cases" page
